advocacy_docs/edb-postgres-ai/analytics/reference/loadingdata.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Loading data (sync or bring your own)
+navTitle: Loading data
+description: How to load data into Lakehouse
+---
+
+## Loading data with Lakehouse sync
+
+If you have a transactional database running in EDB Postgres AI Cloud Service, then you can sync tables from this database into a Managed Storage Location. See ["How to lakehouse sync"](../how_to_lakehouse_sync) for further details.
+
+## Bringing your own data
+
+It's possible to point your Lakehouse node at an arbitrary S3 bucket with Delta Tables inside of it. 
+However, this comes with some major caveats (which will eventually be resolved):
+
+### Caveats
+
+* The tables must be stored as [Delta Lake Tables](http://github.com/delta-io/delta/blob/master/PROTOCOL.md) within the location.
+* A "Delta Lake Table" (or "Delta Table") is a folder of Parquet files along with some JSON metadata.
+* Each table must be prefixed with a `$schema/$table/` where `$schema` and `$table` are valid Postgres identifiers (i.e. < 64 characters)
+  * For example, this is a valid Delta Table that will be recognized by Beacon Analytics:
+    * `my_schema/my_table/{part1.parquet, part2.parquet, _delta_log}`
+      * These `$schema` and `$table` identifiers will be queryable in the Postgres Lakehouse node, e.g.:
+        * `SELECT count(*) FROM my_schema.my_table;`
+  * This Delta Table will NOT be recognized by Postgres Lakehouse node (missing a schema):
+    * `my_table/{part1.parquet, part2.parquet, _delta_log}`
```

Comment:
This is no longer relevant, with `CREATE TABLE` the user can point an external table to any location and name it whatever they want.

```suggestion
```

ex.

```
CREATE TABLE my_schema.my_cool_table () USING PGAA WITH (pgaa.storage_location = 'my-bucket', pgaa.path = 'path/inside/of/bucket/20120101-170000/abcdef/')
```